# Custom message type for prediction data
float32[] box_1
float32[] box_2
float32[] box_3
float32[] box_4
float32 cX
float32 cY
float32 alpha
float32 sigma
float32 sigma_square
float32 sigma_square_log
